lib/events/filelog.go

```diff
@@ -532,6 +541,31 @@ func (l *FileLog) findInFile(fn string, query url.Values, total *int, limit int)
 	return retval, nil
 }
 
+// FilterOutSessionEnd returns whether to filter out session end event
+// determined by whether the input slice contains session end event session ID
+func FilterOutSessionEnd(ef EventFields, r []string) bool {
+	if ef.GetType() == SessionEndEvent {
+		sessionID := ef[SessionEventID].(string)
```

Comment:
Noob question: This type assertion is guaranteed to be valid?

Reply:
I don't believe we assert this to a schema after deserialisation anyway.